packages/custom-webpack/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@angular-builders/custom-webpack",
-  "version": "7.4.2",
+  "version": "7.4.4",
```

Comment:
This is a patch bump, you need to bump a minor version (new feature).

Reply:
No worries, updated to `7.5.0`